models/edr/alerts/alerts_data_monitoring.sql

```diff
@@ -1,11 +1,37 @@
 {{
   config(
-    materialized = 'incremental',
-    unique_key = 'alert_id',
-    on_schema_change = 'append_new_columns'
+    materialized = 'view'
```

Comment:
@oravi 
My only concern here is that the users already have these models as incremental tables...
In our tests we clear the schema, but for them, this will fail... I think.